Makefile

```diff
@@ -36,6 +36,10 @@ functests:
 
 functests-on-ci: setup-test-cluster feature-deploy feature-wait functests
 
+origin-tests:
+	@echo "Running origin-tests"
+	TESTS_TYPE="$(TESTS_TYPE)" hack/run-origin-tests.sh
```

Comment:
TESTS_TYPE is never used
I think you forgot to rename it to `ORIGIN_TESTS_FILTER`